packages/verification-app/src/lib/CertifactionOffchainVerifier.js

```diff
@@ -20,7 +20,7 @@ class CertifactionOffchainVerifier {
                 }
 
                 const fileVerification = {
-                    onBlockchain: data.on_blockchain,
+                    on_blockchain: data.on_blockchain,
```

Comment:
What's the reason for changing it to snake_case?

Reply:
The response from the verify endpoint is snake_case, the goal is to have the same format. I changed this property only because we're using it already and it caused a problem when merging. More camelCase to snake_case changes will be done in the future as well as cleanup deprecated properties.

Reply:
Ok, rest looks good